src/components/autocomplete/js/autocompleteController.js

```diff
@@ -466,9 +466,9 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         select(ctrl.index);
         break;
       case $mdConstant.KEY_CODE.ESCAPE:
+        event.preventDefault(); // Prevent browser from always clearing input
```

Comment:
@topherfangio Yep, the example is in the fixed issue.

There's an older, merged PR that added the ability to instruct ’mdAutocomplete‘ to not clear when you press ESC. The option is ’md-escape-options’. Either Chrome changed their default behavior or somebody changed the ’input’ element type to ’search’ which Chrome clears by default on ESC key down.